package-lock.json

```diff
@@ -4846,7 +4525,7 @@
         },
         "esprima": {
           "version": "https://github.com/ariya/esprima/tarball/master",
-          "integrity": "sha512-W5+ntA5gmzYpBFDpLLw7yAJPohkq5yKF8JnP9UOaXSfnKcibZQcIM9anoxztmB2NwRWLLidEseBbuCdwgQsnig==",
+          "integrity": "sha512-MMxO9L3pF24K8IjZSUFlbLeVmS5bAsXY2sQr9jbV8yw+RseYh3G9yKU1hBHJNb0zfIp/qLIFKV6ZhhqOY4s4VA==",
```

Comment:
Looks like we'll still have the problem even bumping grunt-contrib-jshint since it still pulls in jshint 1.1.0, which has this dependency on esprima master